lang/lang_en_fr.txt

```diff
@@ -894,6 +902,10 @@
 "Please unload the filament first, then repeat this action."
 "SVP, dechargez le filament et reessayez."
 
+# c=20 r=4
+"Please check the IR sensor connection, unload filament if present."
+"SVP, verifiez la connexion du capteur IR et si le filament est decharge."
```

Comment:
I think that this should be `et decharge le filament`("and unload filament", instead of "check if the filament is unloaded")

Reply:
is it shorter than before ;) ?

Reply:
`SVP, verifiez la connexion du capteur IR et decharge le filament.`
this way?

Reply:
Yup!
65 vs 72 characters
```
New
SVP, verifiez la connexion du capteur IR et decharge le filament.
Old
SVP, verifiez la connexion du capteur IR et si le filament est decharge.
```

Reply:
Also used something similar in German translation as well as it makes it shorter and it makes sense
@awenelo Thanks again